### DISCUSSION TITLE
Display voltage and current from motor controller board

### DISCUSSION THREAD

**Discussion by pavelreyesmercado** [NVIDIA-AI-IOT/jetbot#503] [Q&A]
Hi all,
I recently got the Jetbot, booted and ran 'basic motion' and 'teleoperation' notebooks just fine.

Now, I am wondering how to display the voltage and current from the motor controller board in the PiOLED display. I am a completely newbie in Jetbot and Python.

Any help is appreciated.

Thanks!

**Comment by tokk-nv** [comment]
Do you use Waveshare JetBot kit?
If so, you should be able to use their SD card image, and that has the modified code to display the battery voltage.
https://www.waveshare.com/wiki/JetBot_AI_Kit

**Reply by JHPHELAN** [reply]
There is a script stats.py that's supposed to display voltage,  etc.  I tried running this but just ran into trouble and gave up.  Maybe you'll have better luck.

**Reply by rbonari** [reply]
On the waveshare you need to remember to do the update to stats.py as mentioned above. The only time you will see the current is when you plug in the waveshare charger. Then the current display shows the charging current in amps. When the charger is removed for normal operation the current reading on the oled display will always read 0.00 A. The voltage reading shows the battery level.
